mne/io/eyelink/tests/test_eyelink.py

```diff
@@ -224,6 +237,15 @@ def _simulate_eye_tracking_data(in_file, out_file):
                     tokens.append("INPUT")
                 elif event_type == "EBLINK":
                     continue  # simulate no blink events
+                elif event_type == "BUTTON":
+                    # simulate a button event
+                    tokens[1] = "BUTTON"  # simulate button press
+                    tokens[2] = "1"  # simulate button 1
+                    tokens[3] = "1"  # simulate button pressed
+                    tokens[4:4] = ["100", "20", "45", "45", "127.0"]
+                    tokens.extend(["1497.0", "5189.0", "512.5", "............."])
+
+                    continue
```

Comment:
```suggestion
```

Hint: This simulated eyetracking data is built off the actual eyetracking test file ~/mne_data/MNE-testing-data/eyetrack/test_eyelink_HREF.asc, which doesn't have any button events! So the code you added there will never actually be hit : )